src/languages/vbnet.js

```diff
@@ -6,6 +6,26 @@ Website: https://docs.microsoft.com/en-us/dotnet/visual-basic/getting-started/
 */
 
 export default function(hljs) {
+  const commentMode = hljs.COMMENT(
+    '\'',
+    '$',
+    {
+      returnBegin: true,
+      contains: [
+        {
+          className: 'doctag',
+          begin: '\'\'\'|<!--|-->',
```

Comment:
This is confusing, what are the actual variants here?  Do comments sometimes start with `'''` or do they always start with `'` and then later have a `'''` comment inside them, or both?  If possible it'd be nice to flatten this into a more typical `contains` style rule.  Barring that use a look-ahead (there are utils in `regex.js`) to do the match.. `returnBegin` is a bit out of favor now since look-aheads are simpler to use.

 Depending on where this goes doctag might be easier to read as a variant also.

Reply:
I see you just moved it, so if you have no idea what I'm talking about you can leave it as-is. :-)

Reply:
As far as I understand, general comments start with a single `'`, but there are special [Documentation Comments](https://docs.microsoft.com/en-us/dotnet/visual-basic/reference/language-specification/documentation-comments) that start with `'''` and contain XML tags.